signac/synced_collections/buffers/file_buffered_collection.py

```diff
@@ -364,7 +364,7 @@ def buffer_backend(cls, buffer_size=None, force_write=None, *args, **kwargs):
         """
         if force_write is not None:
             warnings.warn(
-                DeprecationWarning(
+                FutureWarning(
```

Comment:
Hmm, something was wrong here in the old code. Can you switch this to call `warnings.warn(message, FutureWarning)` instead of `warnings.warn(FutureWarning(message))`? The function isn't being called correctly.

https://docs.python.org/3/library/warnings.html#warnings.warn

Reply:
Yeah, sure.